numpy/lib/tests/test_io.py

```diff
@@ -1878,49 +1878,52 @@ def test_genfromtxt(self):
             np.savetxt(path, a)
             data = np.genfromtxt(path)
             assert_array_equal(a, data)
-    
+
     @np.testing.dec.skipif(Path is None, "No pathlib.Path")
     def test_ndfromtxt(self):
         # Test outputing a standard ndarray
         with temppath(suffix='.txt') as path:
             path = Path(path)
             with path.open('w') as f:
-                f.write('1 2\n3 4')
+                f.write(u'1 2\n3 4')
```

Comment:
> Attention: this backport module isn’t maintained anymore. If you want to report issues or contribute patches, please consider the pathlib2 project instead.

Says python >= 3.2 recommended. I think you should try with https://pypi.python.org/pypi/pathlib2/